src/guess-the-number.js

```diff
@@ -0,0 +1,26 @@
+const readlineSync = require('readline-sync')
+const prompt = readlineSync.question
+
+const guessTheNumber = guess => {
+  const randomNumber = randomNumberGenerator()
+    if ( guess > randomNumber ) {
```

Comment:
The indentation here makes this code look like it is part of another block within `guessTheNumber` - I actually read it as a function that returned a function until second glance...  Consider indenting appropriately, and adding whitespace between the `randomNumber` declaration, and the body of this function (the `if` statement).